chanzuckerberg/single-cell-data-portal
# feat: add suspension_type to backend

- [335](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell/335)

### Reviewers
**Functional:** 
@ebezzi 
**Readability:** 

---

## Changes
- add suspension_type to database, + corpora and curation API dataset and dataset preview objects
- minor test updates / refactors

## QA steps (optional)
- TODO: Test migration locally
- TODO: Test rdev upload with suspension_type, check that it appears in rdev DB and rdev endpoints

## Notes for Reviewer
- I merged in #3125 in order to get tests to pass (some overlapping necessary changes in there + already has a schema 3.0.0 migration file to use). Look at changes in [02e4d84](https://github.com/chanzuckerberg/single-cell-data-portal/pull/3128/commits/02e4d847c969e9142836ce352828091d285de120) for the bulk of this PR's unique diffs from #3125 onwards
